libp2p/switch.nim

```diff
@@ -329,9 +331,12 @@ proc accept(s: Switch, transport: Transport) {.async.} = # noraises
       debug "Accepted an incoming connection", conn
       asyncSpawn upgradeMonitor(conn, upgrades)
       asyncSpawn s.upgrade.upgradeIncoming(conn)
-    except CancelledError as exc:
+    except CancelledError:
```

Comment:
The `as exc:` is needed for now, as there are multiple bugs surrounding it's lack.